modules/installation-azure-day2-operations-diskencryptionsets.adoc

```diff
@@ -0,0 +1,57 @@
+//Module included in the following assemblies:
+//
+// * installing/installing_azure/enabling-disk-encryption-sets-azure.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="preparing-disk-encryption-sets-day2-operator"]
+= Preparing an Azure Disk Encryption Set for Day2 operator
+The {product-title} installer can use an existing Disk Encryption Set with a user-managed key. To enable this feature, you can create a Disk Encryption Set in Azure and provide the key to the installer. 
+
+.Prerequisite
+
+To enable encryption at host by using system managed key during the installation, set the following fields in `install-config.yaml`:
```

Comment:
I think this should not be the prerequisite for enabling encryptionAtHost on day2, but another way to enable encryptionAtHost by using system managed key during installation.

Reply:
@jinyunma , Can you check if we need to add somehing else for prerequisites?

Reply:
I think nothing to add for prerequisites on OCP side.
Checked in [Azure doc](https://learn.microsoft.com/en-us/azure/virtual-machines/disks-enable-host-based-encryption-portal?tabs=azure-cli#prerequisites), just make sure that the feature "EncryptionAtHost" is enabled for your subscription.